hyperspy/_signals/eels.py

```diff
@@ -1222,13 +1222,13 @@ def kramers_kronig_analysis(self,
         else:
             return eps, output
 
-    def create_model(self, ll=None, auto_background=True, auto_add_edges=True,
+    def create_model(self, low_loss=None, auto_background=True, auto_add_edges=True,
```

Comment:
I agree that `low_loss` is better than `ll`, but this renaming requires its own branch, deprecating `ll` and keeping it until the next major release...